File: crates/deslop-lsp/tests/fixtures/code_action/metrics.applied.py
def extracted_from_cluster_8a7e8e(amounts, tax_rate):
    total = 0
    for amount in amounts:
        taxed = amount * tax_rate // 100
        total = total + amount + taxed
    if total > 10000:
        total = 10000
    return total


def total_with_tax(amounts, tax_rate):
    extracted_from_cluster_8a7e8e(amounts, tax_rate)


def subtotal_with_tax(amounts, tax_rate):
    extracted_from_cluster_8a7e8e(amounts, tax_rate)

File: crates/deslop-lsp/tests/fixtures/code_action/metrics.applied.rs
// TODO: deslop — replace `DeslopTodo` with real types.
type DeslopTodo = ();

fn extracted_from_cluster_d071d5(amounts: DeslopTodo, tax_rate: DeslopTodo) -> DeslopTodo {
    let mut total = 0;
    for amount in amounts {
        let taxed = amount * tax_rate / 100;
        total += amount + taxed;
    }
    if total > 10_000 {
        total = 10_000;
    }
    total
}

pub fn total_with_tax(amounts: &[usize], tax_rate: usize) -> usize {
    extracted_from_cluster_d071d5(amounts, tax_rate)
}

pub fn subtotal_with_tax(amounts: &[usize], tax_rate: usize) -> usize {
    extracted_from_cluster_d071d5(amounts, tax_rate)
}
